frontend/wallet/src/main/App.re

```diff
@@ -53,11 +79,28 @@ let task =
   )
   |> Task.onError(~f=e => Task.succeed(`Error(e)));
 
+module Test = [%graphql {| query { wallets {publicKey} } |}];
+module TestQuery = GraphqlMain.CreateQuery(Test);
+
 Task.perform(
   task,
   ~f=settingsOrError => {
-    // TODO: Send whatever settings are relevant to the relevant pieces
-    createTray(settingsOrError);
+    // TODO: Periodically poll this and/or use a subscription
+    ignore @@
+    Js.Global.setTimeout(
+      () => {
+        let q = TestQuery.query(apolloClient);
```

Comment:
we could do the query in parallel with the settings load, but maybe that will change later?

Reply:
Yeah lets leave it for now. It has to wait for the faker to boot up anyway